src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1010,27 +1011,31 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 		if (items.length === 0) {
 			return;
 		}
+		const fuzzySearchToggle = new Toggle({
+			title: 'Fuzzy search',
+			icon: Codicon.searchFuzzy,
+			isChecked: filterMode === 'fuzzy',
+			inputActiveOptionBorder: this._themeService.getColorTheme().getColor(inputActiveOptionBorder),
+			inputActiveOptionForeground: this._themeService.getColorTheme().getColor(inputActiveOptionForeground),
+			inputActiveOptionBackground: this._themeService.getColorTheme().getColor(inputActiveOptionBackground)
```

Comment:
This is going to be annoying to do for every Toggle. I wish we could pass these colors in via the quickpick construction so we only have to do it once. Is that possible?

Reply:
I tried that and had some issues getting it working. It seems passing it in this way is the standard way of handling it.